internal/clone/registry.go

```diff
@@ -0,0 +1,161 @@
+package clone
+
+import (
+	"bytes"
+	"context"
+	"encoding/gob"
+	"fmt"
+
+	"github.com/Microsoft/hcsshim/internal/regstate"
+	"github.com/Microsoft/hcsshim/internal/uvm"
+)
+
+const (
+	configRoot = "croot"
```

Comment:
`croot` and `ckey` are very non-descriptive. How about `CloneTemplates` and `UVMConfig` instead?

Reply:
I agree that `croot` and `ckey` and not very descriptive. But I think  `configRoot = LateClone` and `configKey = UvmConfig` might be better?